extensions/tailscale/package.json

```diff
@@ -55,6 +55,13 @@
       "subtitle": "Tailscale",
       "description": "Open the Tailscale Admin Console",
       "mode": "no-view"
+    },
+    {
+      "name": "status",
+      "title": "Tailscale Status",
```

Comment:
We can probably also get rid of this "Tailscale" too, since it should already be clear with the icon:

![Screenshot 2023-11-10 at 11 49 01@2x](https://github.com/raycast/extensions/assets/303731/6f284ec0-e6ba-4716-82b6-aa7c136ede8b)